app/javascript/components/pxe-image-type-form/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './pxe-image-type-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const PxeImageForm = ({ recordId }) => {
+  const [{ initialValues, isLoading }, setState] = useState({ isLoading: !!recordId });
+  const submitLabel = !!recordId ? __('Save') : __('Add');
+
+  useEffect(() => {	
+    if (recordId) {
+      miqSparkleOn();
+      API.get(`/api/pxe_image_types/${recordId}`).then(initialValues => {
+          setState({ initialValues, isLoading: false });
+          miqSparkleOff();
+      });
+    }
+  }, [recordId]);
+
+  const onSubmit = ({ edit: _edit, ...values }) => {
```

Comment:
Why the `edit: _edit` and the destruct if you're not using that variable?

```suggestion
  const onSubmit = (values) => {
```